samples/sensor/mcp9808/src/main.c

```diff
@@ -70,8 +135,9 @@ void main(void)
 			break;
 		}
 
-		printf("temp: %d.%06d\n", temp.val1, temp.val2);
+		printf("%s: %g Cel\n", now_str(),
```

Comment:
nit: C instead of Cel, all other samples use C.

Reply:
All other samples are "wrong" because the unit is degrees Celsius, not [coulombs](http://unitsofmeasure.org/ucum.html#section-Base-Units), but I'll change this.

Reply:
:) 
some sample actually have °C, but those have some kind of display.